lib/bender/capistrano/tasks/robots.rake

```diff
@@ -0,0 +1,16 @@
+namespace :deploy do
+  desc "Disallow robots unless deploying to production."
+  task :disallow_robots do
+    return if fetch(:stage) == :production
```

Comment:
This is a rake task block, not a method, could it be that this `return` aborts everything? Or does it actually work as intended?